pset-iterators.js

```diff
@@ -211,6 +241,26 @@ console.log('Problem 8:')
 
 // Add your code below this line
 
+// Still working on this one... ¯\_(ツ)_/¯
```

Comment:
Totally okay to continue working on this problem; you are heading in the right direction for sure! Creating an empty object and using a loop to push in the different key/value pairs is one way to go. [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L244-L260) is an example of that using the reduce method (not to dissimilar to the for loop you're using below).